sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-by-filter/README.md

```diff
@@ -75,7 +75,7 @@ As a first step you'll need to:
    - Click the **Add a permission** button and then,
    - Ensure that the **Microsoft APIs** tab is selected
    - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**
-   - In the **Delegated permissions** section, ensure that the right permissions are checked: **Directory.AccessAsUser.All**
+   - In the **Delegated permissions** section, ensure that the right permissions are checked: **Directory.Read.All**
    - Select the **Add permissions** button ![add permissions](docs/add_permissions.png "add permissions")
    
```

Comment:
`create_secret.png / grant_admin_consent.png / secret_value.png` this file is not used?

Reply:
> `create_secret.png / grant_admin_consent.png / secret_value.png` this file is not used?

They are used and will be kept